src/loaders/genome_attributes/parse_computed_genome_attribs.py

```diff
@@ -0,0 +1,247 @@
+"""
+PROTOTYPE
+
+This script involves processing computed genome attributes and organizing them into a structured format suitable
+for importing into ArangoDB. The resulting JSON file will be used to update (upsert/insert) the database with the parsed
+genome attribute data.
+
+Note: If the ArangoDB collection has been previously created using a JSON file generated by the genome attributes
+      loader script, it is crucial to ensure that the arguments "--load_ver" and "--kbase_collection" are consistent
+      with the ones used in the genome attributes loader script in order to ensure that the same key is generated
+      for the corresponding Arango document."
```

Comment:
Can we add here something like "...and if you want to replace the data created by that loader in ArangoDB, ..."

It should be clear that if you're just doing a regular old load, or you want to leave the gtdb specific data alone, you don't have to care about this

Reply:
👍

Reply:
:+1: